nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeoctl.bat

```diff
@@ -87,9 +87,11 @@ FOR /F "usebackq eol=# tokens=1,* delims==" %%A in ("%NUXEO_CONF%") do (
   if "%%A" == "nuxeo.log.dir" set LOCAL_NUXEO_LOG_DIR=%%B
   if "%%A" == "nuxeo.tmp.dir" set LOCAL_NUXEO_TMP_DIR=%%B
   if "%%A" == "JAVA_OPTS" (
-    set __JAVA_OPTS=%%B
-    REM ***** Expand local string and replace $JAVA_OPTS with %LOCAL_JAVA_OPTS% *****
-    set LOCAL_JAVA_OPTS=!LOCAL_JAVA_OPTS! !__JAVA_OPTS:$JAVA_OPTS=%LOCAL_JAVA_OPTS%!
+    REM ***** Remove word "$JAVA_OPTS" from the line
+    REM ***** And append the remaining values to the existing options
```

Comment:
:-1:  I prefer the original comment which was clearer regarding the goal.
Instead of a functional comment, you describe the technical actions: it is useless (and wrong: << remove word "$JAVA_OPTS" >> while it is << remove "$JAVA_OPTS=" >>).

Reply:
The string replacement in Batch is a little bit tricky, it's syntax is « _variable_:OLD=NEW ». So if we want to replace word _"A"_ by _"B"_ from sentence _"Hello, A"_, we can do:

```batch
SET sentence=Hello, A
SET sentence=%sentence:A=B%
```
If we need to remove _"A"_ from the sentence, then we simply do:

```batch
SET sentence=%sentence:A=%
```

Now, come back to our case, we're handling the Nuxeo configuration file `nuxeo.conf`. We need to:

1. Read the line
2. Extract part B of the line, which is the right side of the first equal sign `=`, store as `%%B`
3. Remove (previously inject) the keyword `$JAVA_OPTS` from the line.

For example, if the line is:

```
JAVA_OPTS=$JAVA_OPTS -Dfile.encoding=UTF-8
```

Then the part B `%%B` is:

```
$JAVA_OPTS -Dfile.encoding=UTF-8
```

This value is set to `NEW_OPTS`. Step 3 removes `$JAVA_OPTS` from `NEW_OPTS` (replaces `$JAVA_OPTS` by empty):

```batch
SET NEW_OPTS=!NEW_OPTS:$JAVA_OPTS=!
```

but we still have a leading space:

```
 -Dfile.encoding=UTF-8
^
```

Reply:
So the comment is correct, the removed word is `$JAVA_OPTS` but not `$JAVA_OPTS=`.